trollsift/parser.py

```diff
@@ -45,8 +45,18 @@ def compose(self, keyvals):
         '''
         return compose(self.fmt, keyvals)
 
+    def partial_compose(self, keyvals):
+        '''Return string composed according to *fmt* string and filled
+        with values with the corresponding keys in *keyvals* dictionary.
+        Not all params in *fmt* need to be specified in *keyvals*.
+        Unspecified parameters are left unchanged.
+        '''
+        return partial_compose(self.fmt, keyvals)
```

Comment:
I know the other methods don't follow this, but could you rewrite this docstring to have a single "subject line" and then a blank line and then the rest:

```python
def partial_compose(self, keyvals):
    """ABC.

    DEFGH.
    """
```

It makes it more inline with what we would like for formatting in the future (once we have the time to force it).

Reply:
Makes sense. I just tried to make it look the same way as the other methods. I'll fix this.

Reply:
Done.